### ISSUE TITLE
Messaging between two bots of the same tenant, is it possible?

### ISSUE THREAD

**Issue by stefan2410** [microsoftgraph/microsoft-graph-comms-samples#466]
Is it possible a calling bot of the same tenant to send json messages to another bot -tabbed application registered in the same tenant? as an incoming webhook for a specific user ?

**Comment by 1fabi0** [comment]
Can you please explain what you mean by that? Because for me it's not completely clear what you mean by this, especially with "for a specific user"-part.

**Comment by stefan2410** [comment]
@1fabi0  like a  bot of the same tenant to send a  json message to a user for  another bot, running in his client.

**Comment by 1fabi0** [comment]
This repository is for real time media bot's with the Graph Media Platform which Teams is currently using for it's meeting capability, of course you can create a connection to your frontend which is dispatching messages. As a side note the c# Skype.Bots.Media Library was original created for policy recording to reliable record meetings but it's way more used due to it's sending media capabilities eg. The together mode of teams meetings uses a Bot in it's backend to merge all the videos

**Comment by stefan2410** [comment]
@1fabi0  I don't think you understand my question.  I know very well what this repository does.

**Comment by 1fabi0** [comment]
So what is this about the js library, I think what you want to do is to have the Skype media bot running on the users site but receiving the roasters on the server but this seems like a bad idea to me because you would have to forward the Auth token of the client app which can do at least the calls to your users, if it's not what you want to do pleas explain me clearly what you want to do

**Comment by stefan2410** [comment]
No I don't want that.  You did not understand me. Skype media bot does his job.  Not as you supposed that I said

**Comment by 1fabi0** [comment]
Ah sorry I didn't see your edit, yes I think I now understand what you mean by that, but I think it's not going to happen in the near future because this will probably require major updates in the platform which rarely done to enhance it with features like this especially since there are things on the queue for the platform which are more commonly asked like receiving sharing of iframes and other things. So probably it's the easiest to use a websocket server for this communication

**Comment by stefan2410** [comment]
It does not concern the media platform .  
>like receiving sharing of iframes and other things

are you related to Microsoft?

**Comment by 1fabi0** [comment]
I had contact with Stephen Sulze and have a bit contact to the official microsoft Graph preview partner channel(or how ever it's called, but it's about graph api news and questions) also I have worked quite a lot with the library

**Comment by stefan2410** [comment]
Thanks for your effort to answer, I appreciate it, but I asked another thing. Probably, I will ask via the partners' channel.

**Comment by 1fabi0** [comment]
Meanwhile you could also try to contact Stephen Sulze but it will probably take quite long until you get an answer. Maybe you find another azure graph service for dispatching the messages, idk

**Comment by adityaramgopal** [comment]
@zhengni-msft do you know who would be able to answer messaging related questions?

**Comment by stefan2410** [comment]
@adityaramgopal   I asked also the same question in the microsoft-teams-library-js repository. Any help is appreciated
`A calling bot registered to the tenant join the calls. If there is another bot - tabbed application in users, would be possible the first bot to send json messages to the user (it knows the user ID and/or UPN), including in the message the AppId of the front-end bot, which could listen in events using the microsoft-teams-library-js, to get the message?`
https://github.com/OfficeDev/microsoft-teams-library-js/issues/608
